prefs.toml

```diff
@@ -41,19 +40,17 @@
    # The Path indicates what value to change, while Value represents the new value. The node operator must make sure
    # to follow the same type of the original value (ex: uint32: 37, float32: 37.0, bool: true)
    # Also, the Value can be a struct (ex: { StartEpoch = 0, Version = "1.5" }) or an array (ex: [{ StartEpoch = 0, Version = "1.4" }, { StartEpoch = 1, Version = "1.5" }])
-   # File represents the file name that holds the configuration. Currently, the supported files are:
-   # api.toml, config.toml, economics.toml, enableEpochs.toml, enableRounds.toml, external.toml, fullArchiveP2P.toml, p2p.toml, ratings.toml, systemSmartContractsConfig.toml
+   # File represents the file name that holds the configuration. Currently, the supported files are: config.toml, external.toml, p2p.toml, enableEpochs.toml and fullArchiveP2P.toml
    # -------------------------------
    # Un-comment and update the following section in order to enable config values overloading
    # -------------------------------
-   # OverridableConfigTomlValues = [
-   # { File = "config.toml", Path = "StoragePruning.NumEpochsToKeep", Value = 4 },
-   # { File = "config.toml", Path = "MiniBlocksStorage.Cache.Name", Value = "MiniBlocksStorage" },
-   # { File = "external.toml", Path = "ElasticSearchConnector.Enabled", Value = true },
-   # { File = "external.toml", Path = "HostDriversConfig", Value = [
-   #    { Enabled = false, URL = "127.0.0.1:22111" },
-   # ] },
-   # ]
+   OverridableConfigTomlValues = [
```

Comment:
All these should be set directly in .toml files.

Reply:
Right, moved all changes in config.toml, except for baseIssueCost, will leave it there as example